code_annotations/exceptions.py

```diff
@@ -7,5 +7,3 @@ class ConfigurationException(Exception):
     """
     Exception specific to code annotation configuration problems.
     """
```

Comment:
Is there any value in this file at all now that it's an empty definition?

Reply:
Yes...various places in the code raise `ConfigurationException` and pass in a specific message.  Callers of those functions can check for this exception specifically, which is better than looking for the generic `Exception`.